plugin/data_masking/include/udf/udf_gen_range.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _PLUGIN_DATAMASKING_GENRANGE_H
+/* Copyright (c) 2018, 2019 Francisco Miguel Biete Banon. All rights reserved.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
+
+#define _PLUGIN_DATAMASKING_GENRANGE_H
+
+#include "plugin/data_masking/include/udf/udf_descriptor.h"
+
+udf_descriptor udf_gen_range();
```

Comment:
I would suggest to aggegrate all the  generate functions (udf_*gen_*()) into one .h and .cc file.

Reply:
It started like that, but it was a huge file. I think it's more easy to maintain and extend if the methods are in different smaller files.